src/Picker/Picker.web.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Icon } from '../Icon';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+import type { Props } from './PickerTypes';
+
+const DEFAULT_OPTION = 'default-empty-option';
```

Comment:
Could you explain what this achieves?

Reply:
Sure. Short description added to const. Hope it will be clearer now.